torch/_inductor/select_algorithm.py

```diff
@@ -842,6 +846,40 @@ def autotune(choices):
             or log.getEffectiveLevel() == logging.DEBUG
         ):
             self.log_results(name, input_nodes, timings, autotune_elapse)
+
+        if return_multi_template:
+            min_extern_choice = float("inf")
+            for choice, timing in timings.items():
+                if isinstance(choice, ExternKernelCaller):
+                    min_extern_choice = min(min_extern_choice, timing)
+
+            timings = {
+                choice: time
+                for choice, time in timings.items()
+                if (
+                    time <= min_extern_choice
+                    or not isinstance(choice, ExternKernelCaller)
+                )
+            }
```

Comment:
hmm, this is no-op? Since we only have one ExternKernelCaller

Reply:
in `tuned_addmm` we select from multiple options